src/applications/vaos/tests/express-care/components/ExpressCareConfirmationPage.unit.spec.jsx

```diff
@@ -45,8 +46,14 @@ const initialState = {
 };
 
 describe('VAOS integration: Express Care form submission', () => {
-  beforeEach(() => mockFetch());
-  afterEach(() => resetFetch());
+  beforeEach(() => {
```

Comment:
I put several of these mock dates in EC because the tests currently break around midnight, if you run them at just the right time.